src/include/sof/audio/coefficients/fft/twiddle_32.h

```diff
@@ -1,29 +1,29 @@
 /* SPDX-License-Identifier: BSD-3-Clause
  *
- * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
  *
  */
 
 /* Twiddle factors in Q1.31 format */
 
-#ifndef __INCLUDE_TWIDDLE_H__
-#define __INCLUDE_TWIDDLE_H__
+#ifndef __INCLUDE_TWIDDLE_32_H__
+#define __INCLUDE_TWIDDLE_32_H__
 
 #include <stdint.h>
 
 #define FFT_SIZE_MAX	1024
 
 /* in Q1.31, generated from cos(i * 2 * pi / FFT_SIZE_MAX) */
-const int32_t twiddle_real[FFT_SIZE_MAX] = {
+const int32_t twiddle_real_32[FFT_SIZE_MAX] = {
 	2147483647,
 	2147443222,
```

Comment:
for i =1, it is cos(2*pi/1024) * 2147483647, is that right? if so, the number should be: 2147483635? anything wrong with my calculation?

Reply:
The computation for twiddle_32.h is here https://github.com/thesofproject/sof/blob/main/tools/tune/fft/export_twiddle.m .

Reply:
Thank you, Seppo, the matlab code seems very nice, I will install matlab later, :)

Reply:
@btian1 I use actually mostly Octave when extra capabilities of Matlab are not needed. On Ubuntu `apt install octave octave-signal octave-io` gives most of needed for audio signal processing.

Reply:
thanks, Seppo, installed, will try to learn for how to use it later.